boom! testing luminance-0.12.1 against nightly for nightly
boom! running: cargo +nightly build --frozen
boom! creating container for: cargo +nightly build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly build --frozen cargobomb`
blam! cffa297d5c0e857aded3b5d45c8bb75a514ed63415a7a71661accdef5111db3b
boom! running `docker start -a cffa297d5c0e857aded3b5d45c8bb75a514ed63415a7a71661accdef5111db3b`
kablam!    Compiling luminance v0.12.1 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 1.14 secs
boom! running `docker rm -f cffa297d5c0e857aded3b5d45c8bb75a514ed63415a7a71661accdef5111db3b`
blam! cffa297d5c0e857aded3b5d45c8bb75a514ed63415a7a71661accdef5111db3b
boom! running: cargo +nightly test --frozen --no-run
boom! creating container for: cargo +nightly test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen --no-run cargobomb`
blam! f5a435f5418eae22cb09b7f4b6fce2113918506c4b119a12439000ff49e2901d
boom! running `docker start -a f5a435f5418eae22cb09b7f4b6fce2113918506c4b119a12439000ff49e2901d`
kablam!    Compiling luminance v0.12.1 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 1.35 secs
boom! running `docker rm -f f5a435f5418eae22cb09b7f4b6fce2113918506c4b119a12439000ff49e2901d`
blam! f5a435f5418eae22cb09b7f4b6fce2113918506c4b119a12439000ff49e2901d
boom! running: cargo +nightly test --frozen
boom! creating container for: cargo +nightly test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen cargobomb`
blam! 4f5054fa813bb15d5c6ec0b09b1ec737b62bcd0eff8a767eb6472e75eebeee20
boom! running `docker start -a 4f5054fa813bb15d5c6ec0b09b1ec737b62bcd0eff8a767eb6472e75eebeee20`
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
kablam!      Running /target/debug/deps/luminance-754b729c06f804a6
kablam!    Doc-tests luminance
blam! 
blam! running 0 tests
blam! 
blam! test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured
blam! 
blam! 
blam! running 10 tests
blam! test src/buffer.rs - buffer (line 39) ... FAILED
blam! test src/buffer.rs - buffer (line 24) ... FAILED
blam! test src/buffer.rs - buffer (line 32) ... FAILED
blam! test src/buffer.rs - buffer (line 48) ... FAILED
blam! test src/buffer.rs - buffer (line 8) ... FAILED
blam! test src/chain.rs - chain (line 30) ... FAILED
blam! test src/shader/program.rs - shader::program (line 37) ... FAILED
blam! test src/chain.rs - chain::Chain (line 18) ... FAILED
blam! test src/texture.rs - texture::Wrap::Repeat (line 16) ... FAILED
blam! test src/texture.rs - texture::Sampler (line 395) ... FAILED
blam! 
blam! failures:
blam! 
blam! ---- src/buffer.rs - buffer (line 39) stdout ----
blam! 	error[E0425]: cannot find value `buffer` in this scope
blam!  --> <anon>:2:1
blam!   |
blam! 2 | buffer.set(3, 3.14);
blam!   | ^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /checkout/src/librustc/session/mod.rs:203
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
blam! ---- src/buffer.rs - buffer (line 24) stdout ----
blam! 	error[E0425]: cannot find value `buffer` in this scope
blam!  --> <anon>:2:1
blam!   |
blam! 2 | buffer.clear(0.);
blam!   | ^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /checkout/src/librustc/session/mod.rs:203
blam! 
blam! ---- src/buffer.rs - buffer (line 32) stdout ----
blam! 	error[E0425]: cannot find value `buffer` in this scope
blam!  --> <anon>:2:1
blam!   |
blam! 2 | buffer.fill([1, 2, 3, 4, 5]);
blam!   | ^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /checkout/src/librustc/session/mod.rs:203
blam! 
blam! ---- src/buffer.rs - buffer (line 48) stdout ----
blam! 	error[E0425]: cannot find value `buffer` in this scope
blam!  --> <anon>:3:17
blam!   |
blam! 3 | let all_elems = buffer.whole();
blam!   |                 ^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `all_elemns` in this scope
blam!  --> <anon>:5:12
blam!   |
blam! 5 | assert_eq!(all_elemns, vec![1, 2, 3, 3.14, 5]); // admit floating equalities
blam!   |            ^^^^^^^^^^ did you mean `all_elems`?
blam! 
blam! error[E0425]: cannot find value `buffer` in this scope
blam!  --> <anon>:8:12
blam!   |
blam! 8 | assert_eq!(buffer.get(3), Some(3.14));
blam!   |            ^^^^^^ not found in this scope
blam! 
blam! error[E0308]: mismatched types
blam!  --> <anon>:5:38
blam!   |
blam! 5 | assert_eq!(all_elemns, vec![1, 2, 3, 3.14, 5]); // admit floating equalities
blam!   |                                      ^^^^ expected integral variable, found floating-point variable
blam!   |
blam!   = note: expected type `{integer}`
blam!              found type `{float}`
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /checkout/src/librustc/session/mod.rs:203
blam! 
blam! ---- src/buffer.rs - buffer (line 8) stdout ----
blam! 	error[E0412]: cannot find type `Buffer` in this scope
blam!  --> <anon>:2:13
blam!   |
blam! 2 | let buffer: Buffer<f32> = Buffer::new(5);
blam!   |             ^^^^^^^^^^^ not found in this scope
blam! 
blam! error[E0433]: failed to resolve. Use of undeclared type or module `Buffer`
blam!  --> <anon>:2:27
blam!   |
blam! 2 | let buffer: Buffer<f32> = Buffer::new(5);
blam!   |                           ^^^^^^^^^^^ Use of undeclared type or module `Buffer`
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /checkout/src/librustc/session/mod.rs:203
blam! 
blam! ---- src/chain.rs - chain (line 30) stdout ----
blam! 	error: cannot find macro `chain!` in this scope
blam!  --> <anon>:3:12
blam!   |
blam! 3 | type Zoo = chain![i32, bool, f32]; // exactly the same type
blam!   |            ^^^^^
blam! 
blam! error[E0412]: cannot find type `Chain` in this scope
blam!  --> <anon>:2:12
blam!   |
blam! 2 | type Foo = Chain<i32,Chain<bool,f32>>;
blam!   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope
blam!   |
blam!   = help: possible candidates are found in other modules, you can import them into scope:
blam!             `use std::io::Chain;`
blam!             `use std::iter::Chain;`
blam! 
blam! error[E0412]: cannot find type `Chain` in this scope
blam!  --> <anon>:2:22
blam!   |
blam! 2 | type Foo = Chain<i32,Chain<bool,f32>>;
blam!   |                      ^^^^^^^^^^^^^^^ not found in this scope
blam!   |
blam!   = help: possible candidates are found in other modules, you can import them into scope:
blam!             `use std::io::Chain;`
blam!             `use std::iter::Chain;`
blam! 
blam! error: aborting due to 3 previous errors
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /checkout/src/librustc_errors/lib.rs:454
blam! thread 'rustc' panicked at 'couldn't compile the test', /checkout/src/librustdoc/test.rs:270
blam! 
blam! ---- src/shader/program.rs - shader::program (line 37) stdout ----
blam! 	error[E0433]: failed to resolve. Use of undeclared type or module `Program`
blam!  --> <anon>:3:15
blam!   |
blam! 3 | let program = Program::new(None, &vs, None, &fs, |get_uni| {
blam!   |               ^^^^^^^^^^^^ Use of undeclared type or module `Program`
blam! 
blam! error[E0425]: cannot find value `vs` in this scope
blam!  --> <anon>:3:35
blam!   |
blam! 3 | let program = Program::new(None, &vs, None, &fs, |get_uni| {
blam!   |                                   ^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `fs` in this scope
blam!  --> <anon>:3:46
blam!   |
blam! 3 | let program = Program::new(None, &vs, None, &fs, |get_uni| {
blam!   |                                              ^^ not found in this scope
blam! 
blam! error[E0412]: cannot find type `Uniform` in this scope
blam!  --> <anon>:4:19
blam!   |
blam! 4 |   let resolution: Uniform<[f32; 2]> = try!(get_uni("resolution"));
blam!   |                   ^^^^^^^^^^^^^^^^^ not found in this scope
blam! 
blam! error[E0412]: cannot find type `Uniform` in this scope
blam!  --> <anon>:5:13
blam!   |
blam! 5 |   let time: Uniform<f32> = try!(get_uni("time"));
blam!   |             ^^^^^^^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /checkout/src/librustc/session/mod.rs:203
blam! 
blam! ---- src/chain.rs - chain::Chain (line 18) stdout ----
blam! 	error: cannot find macro `chain!` in this scope
blam!  --> <anon>:4:12
blam!   |
blam! 4 | type Zoo = chain![i32, bool, f32]; // Zoo == Foo
blam!   |            ^^^^^
blam! 
blam! error[E0412]: cannot find type `Chain` in this scope
blam!  --> <anon>:2:12
blam!   |
blam! 2 | type Foo = Chain<i32,Chain<bool,f32>>;
blam!   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope
blam!   |
blam!   = help: possible candidates are found in other modules, you can import them into scope:
blam!             `use std::io::Chain;`
blam!             `use std::iter::Chain;`
blam! 
blam! error[E0412]: cannot find type `Chain` in this scope
blam!  --> <anon>:2:22
blam!   |
blam! 2 | type Foo = Chain<i32,Chain<bool,f32>>;
blam!   |                      ^^^^^^^^^^^^^^^ not found in this scope
blam!   |
blam!   = help: possible candidates are found in other modules, you can import them into scope:
blam!             `use std::io::Chain;`
blam!             `use std::iter::Chain;`
blam! 
blam! error[E0412]: cannot find type `Chain` in this scope
blam!  --> <anon>:3:12
blam!   |
blam! 3 | type Bar = Chain<Chain<i32, bool>, f32>;
blam!   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ not found in this scope
blam!   |
blam!   = help: possible candidates are found in other modules, you can import them into scope:
blam!             `use std::io::Chain;`
blam!             `use std::iter::Chain;`
blam! 
blam! error[E0412]: cannot find type `Chain` in this scope
blam!  --> <anon>:3:18
blam!   |
blam! 3 | type Bar = Chain<Chain<i32, bool>, f32>;
blam!   |                  ^^^^^^^^^^^^^^^^ not found in this scope
blam!   |
blam!   = help: possible candidates are found in other modules, you can import them into scope:
blam!             `use std::io::Chain;`
blam!             `use std::iter::Chain;`
blam! 
blam! error: aborting due to 5 previous errors
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /checkout/src/librustc_errors/lib.rs:454
blam! thread 'rustc' panicked at 'couldn't compile the test', /checkout/src/librustdoc/test.rs:270
blam! 
blam! ---- src/texture.rs - texture::Wrap::Repeat (line 16) stdout ----
blam! 	error[E0425]: cannot find value `coord_ith` in this scope
blam!  --> <anon>:3:1
blam!   |
blam! 3 | coord_ith = frac(coord_ith); // always between [0;1]
blam!   | ^^^^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find function `frac` in this scope
blam!  --> <anon>:3:13
blam!   |
blam! 3 | coord_ith = frac(coord_ith); // always between [0;1]
blam!   |             ^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `coord_ith` in this scope
blam!  --> <anon>:3:18
blam!   |
blam! 3 | coord_ith = frac(coord_ith); // always between [0;1]
blam!   |                  ^^^^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /checkout/src/librustc/session/mod.rs:203
blam! 
blam! ---- src/texture.rs - texture::Sampler (line 395) stdout ----
blam! 	error[E0422]: cannot find struct, variant or union type `Sampler` in this scope
blam!  --> <anon>:2:1
blam!   |
blam! 2 | Sampler {
blam!   | ^^^^^^^ not found in this scope
blam! 
blam! error[E0433]: failed to resolve. Use of undeclared type or module `Wrap`
blam!  --> <anon>:3:11
blam!   |
blam! 3 |   wrap_r: Wrap::ClampToEdge,
blam!   |           ^^^^^^^^^^^^^^^^^ Use of undeclared type or module `Wrap`
blam! 
blam! error[E0433]: failed to resolve. Use of undeclared type or module `Wrap`
blam!  --> <anon>:4:11
blam!   |
blam! 4 |   wrap_s: Wrap::ClampToEdge,
blam!   |           ^^^^^^^^^^^^^^^^^ Use of undeclared type or module `Wrap`
blam! 
blam! error[E0433]: failed to resolve. Use of undeclared type or module `Wrap`
blam!  --> <anon>:5:11
blam!   |
blam! 5 |   wrap_t: Wrap::ClampToEdge,
blam!   |           ^^^^^^^^^^^^^^^^^ Use of undeclared type or module `Wrap`
blam! 
blam! error[E0433]: failed to resolve. Use of undeclared type or module `Filter`
blam!  --> <anon>:6:17
blam!   |
blam! 6 |   minification: Filter::Linear,
blam!   |                 ^^^^^^^^^^^^^^ Use of undeclared type or module `Filter`
blam! 
blam! error[E0433]: failed to resolve. Use of undeclared type or module `Filter`
blam!  --> <anon>:7:18
blam!   |
blam! 7 |   magnification: Filter::Linear,
blam!   |                  ^^^^^^^^^^^^^^ Use of undeclared type or module `Filter`
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /checkout/src/librustc/session/mod.rs:203
blam! 
blam! 
blam! failures:
blam!     src/buffer.rs - buffer (line 24)
blam!     src/buffer.rs - buffer (line 32)
blam!     src/buffer.rs - buffer (line 39)
blam!     src/buffer.rs - buffer (line 48)
blam!     src/buffer.rs - buffer (line 8)
blam!     src/chain.rs - chain (line 30)
blam!     src/chain.rs - chain::Chain (line 18)
blam!     src/shader/program.rs - shader::program (line 37)
blam!     src/texture.rs - texture::Sampler (line 395)
blam!     src/texture.rs - texture::Wrap::Repeat (line 16)
blam! 
blam! test result: FAILED. 0 passed; 10 failed; 0 ignored; 0 measured
blam! 
kablam! error: test failed
boom! running `docker rm -f 4f5054fa813bb15d5c6ec0b09b1ec737b62bcd0eff8a767eb6472e75eebeee20`
blam! 4f5054fa813bb15d5c6ec0b09b1ec737b62bcd0eff8a767eb6472e75eebeee20
